doc/build.md

```diff
@@ -248,3 +248,18 @@ a) [NIOS2 GNU](https://www.intel.com/content/www/us/en/docs/programmable/683689/
    cmake -DARCH=riscv32 -DTOOLCHAIN=RISCV_XPACK -DTARGET=Release -DCRYPTO=mbedtls ..
    ```
    Note: `make -j` can be used to accelerate the build.
+
+### Linux Builds inside build environments

```

Comment:
I'm not sure. I guess it would also apply to Windows, but I have no way to test it. I also don't know of any build tools that work on Windows, so I'm not sure it's relevant